### PR TITLE
fix(bottomline.phtml): added data-yotpo-section-id field to bottomline widget with proper logic

### PR DESCRIPTION
**Pull request by p-bartosz**

### CODE REVIEW

File: Plugin/AbstractYotpoReviewsSummary.php [hunk 1]
```diff
@@ -48,14 +57,37 @@ public function __construct(
...
             // phpcs:ignore
-            return '<div class="yotpo bottomLine bottomline-position" data-product-id="' . $product->getId() . '" data-url="' . $product->getProductUrl() . '"></div>';
+            return '<div class="yotpo bottomLine bottomline-position"
+                data-product-id="' . $product->getId() . '"
+                data-url="' . $product->getProductUrl() . '"
+            ></div>';
+        }
+    }
+    
+    public static function getWidgetSectionId(string | null $sectionId): string {
```
Comment (ashotnazaryan-yotpo) [inline Plugin/AbstractYotpoReviewsSummary.php:82]: I don't think this is a good place for this function.
Check the implementation of isV3StarRatingWidget function. I think it's better to move your function to that yotpoConfig
`if ($this->_yotpoConfig->isV3StarRatingWidget())`

Comment (p-bartosz) [reply Plugin/AbstractYotpoReviewsSummary.php]: Thanks :)
please take a look now

**Review by lukaszpluto — COMMENTED** [review]
THe only comment is the syntax you use is compatible with Yotpo module minumum PHP version requirements

**Comment by p-bartosz** [comment]
> THe only comment is the syntax you use is compatible with Yotpo module minumum PHP version requirements

done, please check again

File: Plugin/AbstractYotpoReviewsSummary.php [hunk 2]
```diff
@@ -48,14 +48,25 @@ public function __construct(
      * @param Product $product
      * @return string
      */
-    protected function _getCategoryBottomLineHtml(Product $product)
+    protected function _getCategoryBottomLineHtml(Product $product, $templateType)
```
Comment (lukaszpluto) [inline Plugin/AbstractYotpoReviewsSummary.php:51]: maybe add default template type to product

**Review by lukaszpluto — APPROVED** [review]

**Review by rhasson-yotpo — APPROVED** [review]

**Pull request merged**